examples/ui/borders.rs

```diff
@@ -77,228 +46,227 @@ fn setup(mut commands: Commands) {
-            .id();
-        let label_node = commands
-            .spawn((
-                Text::new(label),
-                TextFont {
-                    font_size: 9.0,
-                    ..Default::default()
-                },
-            ))
-            .id();
-        let container = commands
-            .spawn(Node {
-                flex_direction: FlexDirection::Column,
-                align_items: AlignItems::Center,
-                ..default()
-            })
-            .add_children(&[border_node, label_node])
-            .id();
-        commands.entity(root).add_child(container);
-    }
+    let borders_examples = (
+        Node {
+            margin: UiRect::all(Val::Px(25.0)),
+            align_self: AlignSelf::Stretch,
+            justify_self: JustifySelf::Stretch,
+            flex_wrap: FlexWrap::Wrap,
+            justify_content: JustifyContent::FlexStart,
+            align_items: AlignItems::FlexStart,
+            align_content: AlignContent::FlexStart,
+            ..default()
+        },
+        BackgroundColor(Color::srgb(0.25, 0.25, 0.25)),
+        Children::spawn(SpawnIter(border_labels.into_iter().zip(borders).map(
+            |(label, border)| {
+                (
+                    Node {
+                        flex_direction: FlexDirection::Column,
+                        align_items: AlignItems::Center,
+                        ..default()
+                    },
+                    children![
+                        (
+                            Node {
+                                width: Val::Px(50.),
+                                height: Val::Px(50.),
+                                border,
+                                margin: UiRect::all(Val::Px(20.)),
+                                align_items: AlignItems::Center,
+                                justify_content: JustifyContent::Center,
+                                ..default()
+                            },
+                            BackgroundColor(MAROON.into()),
+                            BorderColor(RED.into()),
+                            Outline {
+                                width: Val::Px(6.),
+                                offset: Val::Px(6.),
+                                color: Color::WHITE,
+                            },
+                            children![(
+                                Node {
+                                    width: Val::Px(10.),
+                                    height: Val::Px(10.),
+                                    ..default()
+                                },
+                                BackgroundColor(YELLOW.into()),
+                            )]
+                        ),
+                        (
+                            Text::new(label),
+                            TextFont {
+                                font_size: 9.0,
+                                ..default()
+                            },
+                        )
```

Comment:
Optional suggestion to save a few lines:

```suggestion
                        (Text::new(label), TextFont::from_font_size(9.0),)
```